Pipfile

```diff
@@ -12,7 +12,7 @@ flask = "*"
 flask-cors = "*"
 flask-httpauth = "*"
 gunicorn = "*"
-gevent = {version = "*", platform_python_implementation="=='CPython'"}
+gevent = "*"
```

Comment:
This change was done by the `pipenv upgrade gevent` so I am assuming it's no longer required in the Pipfile. Looking at Github comments it does appear that this was a previous workaround to a previous issue.